tests/_singerlib/test_schema.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
The tests in this module are adapted from https://github.com/transferwise/pipelinewise-singer-python/blob/master/tests/test_schema.py.